synapse/http/server.py

```diff
@@ -573,12 +577,12 @@ def respond_with_json_bytes(
     return NOT_DONE_YET
 
 
-def set_cors_headers(request):
+def set_cors_headers(request: Request):
     """Set the CORs headers so that javascript running in a web browsers can
     use this API
 
     Args:
-        request (twisted.web.http.Request): The http request to add CORs to.
+        request: The http request to add CORs to.
```

Comment:
```suggestion
        request: The http request to add CORS to.
```

while we're here, Cross-Origin Resource Sharing does not have a lowercase 's'.